skt/executable.py

```diff
@@ -220,68 +235,66 @@ def cmd_merge(cfg):
                                        merge_report_path, report_string)
 
                     return
-
                 report_string += '\n  - %s\n    into commit %s' % (mb[0],
                                                                    bhead[:12])
-                idx += 1
-
-        elif cfg.get('patch') or cfg.get('pw'):
-            report_string = 'We applied the following patch(es):'
-
-            if cfg.get('patch'):
-                for patch in cfg.get('patch'):
+                previous_merge = thing_to_merge
+                idx[0] += 1
+
+            elif thing_to_merge[0] == 'patch' or thing_to_merge[0] == 'pw':
+                if previous_merge[0] == 'merge_ref' or not previous_merge[0]:
+                    report_string += '\nWe applied the following patch(es):'
+                if thing_to_merge[0] == 'patch':
+                    patch = thing_to_merge[1]
```

Comment:
This can be joined into a single line with the line below, using `thing_to_merge[1]` as the argument

Reply:
I fix it